### ISSUE TITLE
Pagerfanta compatibility with Sylius 1.2.4

### ISSUE THREAD

**Issue by F-Locos** [BitBagCommerce/SyliusElasticsearchPlugin#23]
Hi,

I found a compatibility problem between Sylius and ElasticsearchPlugin.
It's with the 'Pagerfanta' bundle which make the problem.

So if you use Sylius v1.2.4, and want to use ElasticSearch plugin, you need to : 
- either, downgrade the version of Sylius to 1.0 (👎 )
- either, downgrade 'Pagerfanta' with the command : composer update pagerfanta/pagerfanter (it will update to downgrade it automatically 👍 )

Hope you can fix it, good luck and good job !

**Comment by spras** [comment]
Seems fixed in https://github.com/FriendsOfSymfony/FOSElasticaBundle/pull/1427 but not released yet in elastica bundle :-( 

i could install syliusElasticBundle like this :
```
composer require friendsofsymfony/elastica-bundle dev-master
composer require bitbag/elasticsearch-plugin
```
Hope it can help

**Comment by bitbager** [comment]
Thank you @spras. Are there any updates on this?

**Comment by coordtechjetpulp** [comment]
@bitbager :  No update, the stable version of elastica-bundle is steal 5.0.3, see https://packagist.org/packages/friendsofsymfony/elastica-bundle 
The fix in elastica-bundle is only present in 5.1.*-dev 
:-(

**Comment by kortwotze** [comment]
Looks like its installable with composer now :)
